ui.tests/test-module/specs/wizard/wizard.runtime.repeatability.cy.js

```diff
@@ -232,7 +269,7 @@ describe('visibility of navigation buttons', function () {
         // check when new instance of repeatable panel is added
         // and that new instance gets active & prev and next nav buttons are visible
         cy.get(`#${textInputId}`).find(driverTextInput).focus().clear().type('a').blur().then(() => {
-            getWizardPanelAtIndex(2).should('have.class', wizardPanelActive);
+            getWizardPanelAtIndex(11).should('have.class', wizardPanelActive);
```

Comment:
When there is no change in collateral, why is this test case changed ?

Reply:
It was failing on my local, because the panel at index 11 was active, and it was checking at index 2.

Reply:
The test case was working earlier, ideally if an old test case is changed, it is most like due to collateral else this could be a regression. please investigate

Reply:
Done, reverted to old scenario!